salt/modules/yumpkg.py

```diff
@@ -1159,7 +1159,19 @@ def _add_common_args(cmd):
 
     errors = []
 
+    hold_pkgs = list_holds(full=False)
+    log.error(hold_pkgs)
+    to_hold = []
     if targets:
+        if len(targets) == 1:
+            to_hold.append(pkgname)
```

Comment:
No, there is an else statement. Since when target format is not the same when there is one package that when there is multiple packages, we need to treat that in differents ways.  I think that is the idea here. I need to refresh my memory, since I did this a few weeks ago.